### DISCUSSION TITLE
Windows-like "Show desktop" button

### DISCUSSION THREAD

**Discussion by KhriszB** [lawand-dot-io/taskbar#53] [Feature requests]
As the title say. I would like to have a dedicated button for it. Im not a "hot-key person" and I also use this button on Win and Linux too. 
Thank you for this awesome app BTW!

**Comment by lawand-dot-io** [comment]
Hello I'm glad you like Taskbar. Thanks for the request. This is a duplicate of https://github.com/lawand-dot-io/taskbar/discussions/25. Feel free to upvote it to increase the chance that it gets implemented. Cheers
